gcc/rust/backend/rust-compile-expr.cc

```diff
@@ -1315,12 +1315,8 @@ CompileExpr::visit (HIR::MethodCallExpr &expr)
 				    expr.get_locus ());
     }
   else
-    {
-      // lookup compiled functions since it may have already been compiled
-      HIR::PathExprSegment method_name = expr.get_method_name ();
-      HIR::PathIdentSegment segment_name = method_name.get_segment ();
-      fn_expr = resolve_method_address (fntype, receiver, expr.get_locus ());
-    }
+    // lookup compiled functions since it may have already been compiled
+    fn_expr = resolve_method_address (fntype, receiver, expr.get_locus ());
```

Comment:
hmmm, I feel like there used to be/should be a method cache lookup here which does not exist for some reason? maybe it would be better to comment out the two objects and add a `FIXME` comment?

Reply:
oh ok, fixme comment something like this?
```
// FIXME : method cache lookup
// comment deleted lines
// here
code...
```

Reply:
yes something like this would be good :) but could you investigate that piece of code? was there method lookup at some point here that was removed, or was it just never added?

Reply:
I checked with git blame, what I found was these 2 objects were passed to the below function first, but later function didn't need these parameters, and the above 2 lines were left unchanged

Reply:
i asked @philberty about these 2 lines, he told me to go for it
Philip, do you remember what Arthur is talking about?

Reply:
that's fine then, we can go ahead and remove them :D thanks for checking!